src/pages/resultsView/oncoprint/TracksMenu.tsx

```diff
@@ -60,9 +67,34 @@ export const COUNT_PADDING_WIDTH = 17;
 export default class TracksMenu extends React.Component<IAddTrackProps, {}> {
     @observable tabId: Tab | string = Tab.CLINICAL;
 
+    private genericAssayEntityOptionsReaction: IReactionDisposer;
+
+    private selectedGenericAssayProfileIdByType = observable.map<
+        string,
+        string
+    >({}, { deep: true });
+    @observable XvsYSelection: {
+        x?: { value: string; label: string };
+        y?: { value: string; label: string };
+    } = {
+        x: undefined,
+        y: undefined,
+    };
+
     constructor(props: IAddTrackProps) {
         super(props);
         makeObservable(this);
+        // Using the first profile as default profile option to initialize genericAssayProfileOptionsByType
```

Comment:
Maybe change text to this?
```suggestion
        // Using the first profile as default profile option to initialize selectedGenericAssayProfileIdByType
```